authenticator/customauthenticator/custom.py

```diff
@@ -134,8 +134,8 @@ def check_jwt_token(self, access_token):
             user_roles = []
 
         # check authorization
-        if "incore_jupyter" not in user_groups and "incore_jupyter" not in user_roles:
-            raise web.HTTPError(403, log_message="The current user does not belongs to incore jupyter lab group and " +
+        if "incore_user" not in user_groups and "incore_user" not in user_roles:
```

Comment:
would it make sense to make this an environment variable so we can easily change it in the future?

Reply:
okay... I will also update the incore-helm for this as well